### PR TITLE
HotkeyMessages: Add more quick message slots

### PR DESCRIPTION
**Pull request by tetektoza**
Resolves: https://github.com/diasurgical/devilutionX/issues/6542

Currently we only have 4 quick message slots - this patch adds more of them, resizing the total amount to 10.

Issue #6542 also mentioned number pad setup, so this PR also adds support to bind hotkeys to numbers on keypad (0-9).

### CODE REVIEW

File: Source/diablo.cpp [hunk 1]
```diff
@@ -1766,12 +1772,12 @@ void InitKeymapActions()
...
 	    nullptr,
 	    CanPlayerTakeAction);
-	for (int i = 0; i < 4; ++i) {
+	for (size_t i = 0; i < QUICK_MESSAGE_OPTIONS; ++i) {
 		sgOptions.Keymapper.AddAction(
 		    "QuickMessage{}",
 		    N_("Quick Message {}"),
 		    N_("Use Quick Message in chat."),
-		    SDLK_F9 + i,
+		    SDLK_KP_0 + i,
```
Comment (StephenCWills) [inline Source/diablo.cpp:1780]: `SDLK_KP_0` comes after `SDLK_KP_9` in SDL2.

Comment (StephenCWills) [reply Source/diablo.cpp:1780]: I suppose I should also point out that our default keybinds are set up to closely match the vanilla's keybinds. So the correct default would be to map the first four quick messages to F9-F12 and then leave the rest of them unmapped.

File: Source/diablo.cpp [hunk 2]
```diff
@@ -126,10 +126,16 @@ std::vector<std::string> DebugCmdsFromCommandLine;
...
+	{ "QuickMessage1", N_("I need help! Come here!") },
 	{ "QuickMessage2", N_("Follow me.") },
 	{ "QuickMessage3", N_("Here's something for you.") },
-	{ "QuickMessage4", N_("Now you DIE!") }
+	{ "QuickMessage4", N_("Now you DIE!") },
+	{ "QuickMessage5", N_("Heal yourself!") },
+	{ "QuickMessage6", N_("Watch out!") },
+	{ "QuickMessage7", N_("Thanks.") },
+	{ "QuickMessage8", N_("Retreat!") },
+	{ "QuickMessage9", N_("Sorry..") },
```
Comment (StephenCWills) [inline Source/diablo.cpp:137]: Did you mean to put two periods after "Sorry" here?

File: Source/diablo.cpp (hunk 1 above)
Comment (tetektoza) [reply Source/diablo.cpp]: Oh right, what is interesting is I've found that SDL2 goes with SDLK_KP_0 after SDLK_KP_9, and you can see me applying this logic in `options.cpp`, then I've went back to this and it turns out it works either way for some reason, so I left it out as it is, but anyways - I've changed this.

File: Source/diablo.cpp (hunk 2 above)
Comment (tetektoza) [reply Source/diablo.cpp]: Yes, but I've changed it to single period since it may be more natural.

**Review by StephenCWills — APPROVED** [review]

**Comment by AJenbo** [comment]
nice with defaults an everything :)

**Pull request merged**